src/components/dataDisplays/SightingsDisplay.jsx

```diff
@@ -32,58 +30,56 @@ export default function SightingsDisplay({ sightings, loading }) {
 
   const columns = [
     {
-      name: 'id',
-      label: intl.formatMessage({
-        id: 'SIGHTING_ID',
-      }),
+      name: 'time',
+      labelId: 'SIGHTING_TIME',
       options: {
-        customBodyRender: value => (
-          <Link href={`/sightings/${value}`}>{value}</Link>
-        ),
+        cellRenderer: cellRendererTypes.specifiedTime,
       },
     },
     {
-      name: 'individuals',
-      label: intl.formatMessage({ id: 'INDIVIDUALS' }),
-      options: {
-        customBodyRender: individuals => (
-          <span>
-            {individuals.map((id, i) => (
-              <span key={id}>
-                {i !== 0 && ', '}
-                <Link href={`/individuals/${id}`}>{id}</Link>
-              </span>
-            ))}
-          </span>
-        ),
-      },
+      name: 'locationId_value',
+      labelId: 'REGION',
+      align: 'left',
     },
     {
-      name: 'sightingDate',
-      label: intl.formatMessage({ id: 'SIGHTING_DATE' }),
+      name: 'owners',
+      labelId: 'REPORTER',
+      align: 'left',
       options: {
-        cellRenderer: cellRendererTypes.date,
-        cellRendererProps: { accessor: 'lastSeen' },
+        customBodyRender: owners => {
+          const ownerName = get(
+            owners,
+            [0, 'full_name'],
+            'Unknown User',
+          );
+          const ownerGuid = get(owners, [0, 'guid'], 'Unknown User');
```

Comment:
Do you want a potential link to `/users/Unknown%20User`? It would be cleaner to not return a link for a user without a guid.

Reply:
Ooops copypaste mistake!